### PR TITLE
Fix: PureLayout removal - LaunchImageViewController

### PR DESCRIPTION
**Pull request by billypchan**
## What's new in this PR?

Remove PureLayout from LaunchImageViewController and create snapshot tests.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/LaunchImage/LaunchImageViewController.swift [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    /// Convinience method for showing the @c activityIndicator and @c loadingScreenLabel and start the spinning animation
+    func showLoadingScreen() {
+        shouldShowLoadingScreenOnViewDidLoad = true
+        loadingScreenLabel.isHidden = false
+        activityIndicator.startAnimation(nil)
+    }
+
+    /// Convinience method for hiding all the animation related functionality
+    func hideLoadingScreen() {
+        activityIndicator.stopAnimation(nil != nil)
```
Comment (vytis) [inline Wire-iOS/Sources/UserInterface/LaunchImage/LaunchImageViewController.swift]: What is the reason behind this check? Can't we just use `false`?

File: Wire-iOS/Sources/UserInterface/LaunchImage/LaunchImageViewController.swift [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    private var activityIndicator: ProgressSpinner!
+
+    /// Convinience method for showing the @c activityIndicator and @c loadingScreenLabel and start the spinning animation
+    func showLoadingScreen() {
+        shouldShowLoadingScreenOnViewDidLoad = true
+        loadingScreenLabel.isHidden = false
+        activityIndicator.startAnimation(nil)
+    }
+
+    /// Convinience method for hiding all the animation related functionality
```
Comment (vytis) [inline Wire-iOS/Sources/UserInterface/LaunchImage/LaunchImageViewController.swift]: Typo - should be "Convenience". Another occurrence is in a method above.

File: Wire-iOS/Sources/UserInterface/LaunchImage/LaunchImageViewController.swift (hunk 1 above)
Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/LaunchImage/LaunchImageViewController.swift]: See translation leave over. I will fix it.

File: Wire-iOS/Sources/UserInterface/LaunchImage/LaunchImageViewController.swift (hunk 2 above)
Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/LaunchImage/LaunchImageViewController.swift]: Also some copy & paste error from @objc version :)

**Pull request merged**